src/frontend/src/lib/components/hero/about/AboutHowModal.svelte

```diff
@@ -20,9 +20,9 @@
 	</svelte:fragment>
 
 	<div class="stretch pt-4">
-		<ImgBanner src={CoverHow} alt={$i18n.about.how.text.title} />
+		<ImgBanner size="small" src={CoverHow} alt={$i18n.about.how.text.title} />
 
-		<p class="mt-6">
+		<p class="mt-12">
```

Comment:
Did you change this class to provide the same visual padding as previously?

Reply:
yes

Reply:
I understand. Slowing down network speed. Looks good to me. We should/could make the Img banner take in a class or at least a height instead of having sizes to be more flexible. what do you think?
https://github.com/user-attachments/assets/9d22343b-c17a-43c8-8e0e-27a67c344519